src/user/login.js

```diff
@@ -0,0 +1,136 @@
+import { useContext } from "react";
+import { AuthContext } from "../context/authContext";
+import { useForm } from "react-hook-form";
+import App from "../App";
+export default function Login() {
+
+    const { userExist, passVisible, wrongPassword, setPassVisible, newUser, setNewUser, loginEmail, setLoginEmail,
+        loginPassword, loginAsGuest, user, setRememberMe, login, registerUser } = useContext(AuthContext)
+
+    const {
+        register,
+        handleSubmit,
+        watch,
+        formState: { errors }
+    } = useForm();
+
+    const signUpUser = (data) => {
+        registerUser(data.userName, data.registerEmail, data.registerPassword)
+    };
+
+    const signInUser = (data) => {
+        login(data.userEmail, data.userPassword)
+    };
+    console.log(user, !user);
```

Comment:
remove console.log

Reply:
thank you for the review Abhi, I'll make the changes.